udata/harvest/backends/dcat.py

```diff
@@ -214,7 +221,10 @@ def inner_process_dataset(self, item: HarvestItem, page_number: int, page: Graph
 
         dataset = self.get_dataset(item.remote_id)
         remote_url_prefix = self.get_extra_config_value("remote_url_prefix")
-        return dataset_from_rdf(page, dataset, node=node, remote_url_prefix=remote_url_prefix)
+        dataset = dataset_from_rdf(page, dataset, node=node, remote_url_prefix=remote_url_prefix)
+        if dataset.organization:
+            self.organizations_to_update.add(dataset.organization)
```

Comment:
```suggestion
            dataset.organization.do_not_compute_metrics = true
            self.organizations_to_update.add(dataset.organization)
```

Reply:
I'm worried we're introducing a state that will need to be managed afterward. We should at least reset it at the end of the run. And what if the harvester aborts before state is reset?